src/tests/test_claim_detector.py

```
@@ -0,0 +1,133 @@
+    with pytest.raises(ValidationError) as excinfo:
+        Claim(**invalid_data)
+    assert "rank" in str(excinfo.value) # Check that the error message mentions 'rank'
+
+# --- Tests for the claim_detector_agent Instance ---
+
+def test_claim_detector_agent_instantiation():
+    """Test that the claim_detector_agent is instantiated and has basic properties."""
+    assert claim_detector_agent is not None
+    assert claim_detector_agent.name == "ClaimDetector"
+    
+    assert claim_detector_agent.output_type.__origin__ == list # Checks if it's a list type
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)